src/overflow.rs

```diff
@@ -430,8 +430,8 @@ impl<'a> Context<'a> {
         };
 
         if let Some(rewrite) = rewrite {
-            // splitn(2, *).next().unwrap() is always safe.
-            let rewrite_first_line = Some(rewrite.splitn(2, '\n').next().unwrap().to_owned());
+            // split(*).next().unwrap() never panics.
+            let rewrite_first_line = Some(rewrite.split('\n').next().unwrap().to_owned());
```

Comment:
Is it split_once instead of split?

Reply:
I can also use `split_once` but that's longer
```rust
let rewrite_first_line = Some(
    rewrite
        .split_once('\n')
        .map_or(&*rewrite, |(f, _)| f)
        .to_owned(),
);
```